### PR TITLE
Adds in the shipping forecast

### LINKED ISSUE #2: Add a shipping forecast
Add a new file that contains the shipping forecast.

### PR DESCRIPTION
**Pull request by ukmo-juan-castillo**
Closes #2

### CODE REVIEW

File: shipping-forecast.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (caroduro) [inline shipping-forecast.md:1]: No need of a new line

**Review by caroduro — CHANGES_REQUESTED** [review]
Changes requested

File: shipping-forecast.md (hunk 1 above)
Comment (caroduro) [reply shipping-forecast.md:1]: i mean empty line.

**Review by ukmo-juan-castillo — COMMENTED** [review]
Please review accepted changes.

**Review by caroduro — APPROVED** [review]

**Pull request merged**